_releases/2022-10-06-1.6.0-released.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Crystal 1.6.0 is released!
+version: 1.6.0
+summary:
+thumbnail: +
+author:
+---
+
+We are delivering a new release with several bugfixes and improvements. Below we list the most important or interesting changes, without mentioning several bugfixes and smaller enhancements. For more details, visit the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.6.0). Breaking changes are marked with ⚠️.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.6.0) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+## Stats
+
+In this release we included [183 changes since the 1.5.1 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.6.0) by 26 contributors. We thank all the effort put into improving the language! ❤️
+
+Below we list the most remarkable changes in the language and stdlib.
+
+## ⚠️ Refactor in console methods
+
+By making some console methods available in Windows, a significant [refactor](https://github.com/crystal-lang/crystal/pull/12352) lead to deprecating three macro methods in `FileDescriptor`: `cooked_from_tc_mode!`, `noecho_from_tc_mode!`, and `raw_from_tc_mode!`. Additionally, methods `#noecho!` and `#raw!` now return `nil`.
```

Comment:
I don't think this is actually a very relevant change for users. 
Might make sense to mention it because of the deprecation, but it should be down at the bottom, not the very first item.

Reply:
Actually, given that we found a shard that broke because of the one below, let me keep them on top.